docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,284 @@
+for the config mode packages, but not for the new status page.
+
+i18n namespaces
+===============
+
+In earlier version of Gluon, all gluon-web (formerly LuCI) packages shared the same i18n namespace,
+so independent packages could override each others translations (with an arbitrary translation of
+the same string "winning"). This issue has been solved by giving each package its own translation
+namespace, which is defined by the *package* directive in a package's controller. It is still
+possible to access a different i18n namespace (e.g. gluon-web base or site translations) as
+described in :doc:`../dev/web/i18n`.
+
+Package Makefile cleanup
+========================
+
+The Makefiles of the individual Gluon packages have been cleaned up significantly by moving a
+lot of boilerplate code to *package/gluon.mk*. The new features of *package/gluon.mk* are
+explained in detail in :doc:`../dev/packages`.
+
+Site checker
+============
+
+* New JSON/Lua path specification
+
+  The old string-based path specifications in site check scripts (e.g. ``'autoupdater.branch'``)
+  have been replaced with arrays (``{'autoupdater', 'branch'}``). This will implicitly ensure that
+  *auoupdater* is a table when it exists (simplifying checks for deep structures), and makes it easier
+  to specify paths with variable components (by referencing a variable as an array element).
+
+* Alternatives
+
+  The site check library has gained support for *alternatives*. It is now possible to check
+  if a configuration satisfies one of multiple checks:
+
+  .. code-block:: lua
+
+    -- This can be a boolean or a string!
+    alternatives(function()
+      need_boolean('foo')
+    end, function()
+      need_string('foo')
+    end)
+
+  As many branches (functions) as necessary can be passed to a single *alternatives* call, which will succeed when
+  at least one of the branches succeeds.
+
+Known issues
+************
+
+* Default TX power on many Ubiquiti devices is too high, correct offsets are unknown (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
+
+  Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promicious mode is disallowed).
+
+* Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+* Frequent reboots due to out-of-memory or high load due to memory pressure on weak hardware in larger meshes
+  (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
+
+  Optimizations in Gluon 2018.1 have significantly improved memory usage; still,
+  large networks may experience problems.
```

Comment:
maybe the sentence should be adjusted:

1. because this does not only affect large networks
2. because there will never be a final solution, very big (growing) networks will always have a problem, the bigger the network the bigger the problem

Reply:
I have reworded this a bit.